h/views/activity.py

```diff
@@ -154,6 +154,29 @@ def user_annotation_count(aggregation, userid):
         result['stats'] = {
             'annotation_count': group_annotation_count,
         }
+
+        if self.group.type == 'private':
+            result['group_users_display_info'] = {
```

Comment:
Instead of fully switching on specific group type here, perhaps consider creating a default dictionary that is then modified for different group types. That way a) you won't have to repeat common values and b) there won't be the risk of no group type matching.

Reply:
Also, @hmstepanek 's earlier comment about possibly putting this logic in a private method in the view might be something to consider, too!